cmd/panfs.go

```diff
@@ -1480,7 +1480,7 @@ func (fs *PANFSObjects) DeleteObject(ctx context.Context, bucket, object string,
 		noLock := ""
 		err = fs.configAgent.DeleteObject(objectName, noLock)
 	} else {
-		err = fsDeleteFile(ctx, pathJoin(bucketDir), pathJoin(bucketDir, object))
```

Comment:
Please, add a comment explaining why this change was necessary.

Reply:
By changing `fsDeleteFile()` to `fsRemoveFile()` we change the behavior of MinIO. Previously all the ancestor directories of the removed object were removed if empty up to the root of the bucket.

After the change, the ancestor directories are left untouched. This means after a number of add object - remove object operations we can be left with a bunch of empty directories.

@yauhen-matsiushonak is this an acceptable change in behavior?

Reply:
@lech-lorens-epam , sorry for that - I pushed wrong change by mistake. Updated with the right one - please take look again